core/db/db_test.go

```diff
@@ -31,246 +31,104 @@ func TestMain(m *testing.M) {
 	defer func() {
 		columnfamilies = oldcfs
 	}()
-	db, err = openDB()
-	if nil != err {
-		t.Fatalf("Error re-opening DB with upgraded columnFamilies")
-	}
-	db.CloseDB()
-}
-
-func TestDeleteState(t *testing.T) {
-	testDBWrapper := NewTestDBWrapper()
-	testDBWrapper.CreateFreshDB(t)
-	openchainDB := GetDBHandle()
-	defer testDBWrapper.cleanup()
-	openchainDB.Put(openchainDB.StateCF, []byte("key1"), []byte("value1"))
-	openchainDB.Put(openchainDB.StateDeltaCF, []byte("key2"), []byte("value2"))
-	openchainDB.DeleteState()
-	value1, err := openchainDB.GetFromStateCF([]byte("key1"))
-	if err != nil {
-		t.Fatalf("Error getting in value: %s", err)
-	}
-	if value1 != nil {
-		t.Fatalf("A nil value expected. Found [%s]", value1)
-	}
-
-	value2, err := openchainDB.GetFromStateCF([]byte("key2"))
-	if err != nil {
-		t.Fatalf("Error getting in value: %s", err)
-	}
-	if value2 != nil {
-		t.Fatalf("A nil value expected. Found [%s]", value2)
-	}
-}
-
-func TestDBSnapshot(t *testing.T) {
-	testDBWrapper := NewTestDBWrapper()
-	testDBWrapper.CreateFreshDB(t)
-	openchainDB := GetDBHandle()
-	defer testDBWrapper.cleanup()
-
-	// write key-values
-	openchainDB.Put(openchainDB.BlockchainCF, []byte("key1"), []byte("value1"))
-	openchainDB.Put(openchainDB.BlockchainCF, []byte("key2"), []byte("value2"))
-
-	// create a snapshot
-	snapshot := openchainDB.GetSnapshot()
-
-	// add/delete/modify key-values
-	openchainDB.Delete(openchainDB.BlockchainCF, []byte("key1"))
-	openchainDB.Put(openchainDB.BlockchainCF, []byte("key2"), []byte("value2_new"))
-	openchainDB.Put(openchainDB.BlockchainCF, []byte("key3"), []byte("value3"))
-
-	// test key-values from latest data in db
-	v1, _ := openchainDB.GetFromBlockchainCF([]byte("key1"))
-	v2, _ := openchainDB.GetFromBlockchainCF([]byte("key2"))
-	v3, _ := openchainDB.GetFromBlockchainCF([]byte("key3"))
-	if !bytes.Equal(v1, nil) {
-		t.Fatalf("Expected value from db is 'nil', found [%s]", v1)
-	}
-	if !bytes.Equal(v2, []byte("value2_new")) {
-		t.Fatalf("Expected value from db [%s], found [%s]", "value2_new", v2)
-	}
-	if !bytes.Equal(v3, []byte("value3")) {
-		t.Fatalf("Expected value from db [%s], found [%s]", "value3", v3)
-	}
-
-	// test key-values from snapshot
-	v1, _ = openchainDB.GetFromBlockchainCFSnapshot(snapshot, []byte("key1"))
-	v2, _ = openchainDB.GetFromBlockchainCFSnapshot(snapshot, []byte("key2"))
-	v3, err := openchainDB.GetFromBlockchainCFSnapshot(snapshot, []byte("key3"))
-	if err != nil {
-		t.Fatalf("Error: %s", err)
-	}
-
-	if !bytes.Equal(v1, []byte("value1")) {
-		t.Fatalf("Expected value from db snapshot [%s], found [%s]", "value1", v1)
-	}
-
-	if !bytes.Equal(v2, []byte("value2")) {
-		t.Fatalf("Expected value from db snapshot [%s], found [%s]", "value1", v2)
-	}
-
-	if !bytes.Equal(v3, nil) {
-		t.Fatalf("Expected value from db snapshot is 'nil', found [%s]", v3)
-	}
-}
+	openchainDB = GetDBHandle()
 
-func TestDBIteratorAndSnapshotIterator(t *testing.T) {
```

Comment:
why do you want to delete this test? I had added this unit test for increasing the code coverage during unit tests.